### PR TITLE
TELCODOCS-1241: Remove biosConfigRef from SNO SiteConfig

### PR DESCRIPTION
**Pull request by amolnar-gh**
Version(s): 4.13, 4.14

Issue: https://issues.redhat.com/browse/TELCODOCS-1241

Link to docs preview: https://65101--docspreview.netlify.app/openshift-enterprise/latest/scalability_and_performance/ztp_far_edge/ztp-manual-install.html#ztp-generating-install-and-config-crs-manually_ztp-manual-install

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by mpmaruthu** [comment]
Sign-off from Telco QE: Looks good to me.
/Peri M

File: snippets/ztp_example-sno.yaml [hunk 1]
```diff
@@ -43,8 +43,6 @@ spec:
     nodes:
       - hostName: "example-node1.example.com"
         role: "master"
```
Comment (dfitzmau) [inline snippets/ztp_example-sno.yaml:45]: Please feel free to ignore this comment, it necessary, but should "master" be changed to another term based on https://redhat-documentation.github.io/supplementary-style-guide/#_master_and_slave . Maybe it's a coded name and it cannot be changed. Perhaps an example would be a good replacement, such as `<primary_role_name>`?

**Review by dfitzmau — COMMENTED** [review]
/remove-label peer-review-in-progress

/label peer-review-done

File: snippets/ztp_example-sno.yaml (hunk 1 above)
Comment (amolnar-rh) [reply snippets/ztp_example-sno.yaml:45]: I'll check with the SMEs, but I'll request a merge for this PR as it is out of scope for this issue. Good catch!

**Pull request merged**